package.json

```diff
@@ -10,7 +10,7 @@
         "lint": "eslint . --ignore-path .gitignore",
         "fix": "eslint . --ignore-path .gitignore --fix",
         "travis": "CI=true react-scripts test --env=jsdom -u  --colors",
-        "coverage": "react-scripts test --coverage --env=jsdom -u --colors"
+        "coverage": "react-scripts test --coverage --env=jsdom -u --colors --coverage --coveragePathIgnorePatterns=src/index.js"
```

Comment:
I see duplication here --coverage is appearing twice in the same statement please resolve that